[v1.5][F14-B2] Medeina

Aquarinte opened the pull request:
@chialejing @wynonaK @Robert-Peng

cjianhui commented on README.adoc:
```diff
@@ -1,9 +1,8 @@
 = Address Book (Level 4)
```
Do remember to update this to your product name!

cjianhui commented on README.adoc:
```diff
@@ -1,9 +1,8 @@
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
+https://travis-ci.org/CS2103JAN2018-F14-B2/main[image:https://travis-ci.org/CS2103JAN2018-F14-B2/main.svg?branch=master[Build Status]]
+https://coveralls.io/github/se-edu/CS2103JAN2018-F14-B2/main?branch=master[image:https://coveralls.io/repos/github/CS2103JAN2018-F14-B2/main/badge.svg?branch=master[Coverage Status]]
```
Good job on updating Travis and Coveralls to point to your repo!

cjianhui commented on docs/DeveloperGuide.adoc:
```diff
@@ -800,15 +803,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |user |manage the "rooms" in hospital |make space for newly hospitalized pet patient
+
+|`*` |user with many pet patients in the address book |sort pet patients by name |locate a pet patient easily
```
Good job on coming up with a variety of user stories! It does seem like they match the needs of your target user profile. However, you may want to be more specific on the type of user involved for each user story. Do update the `user` in the user stories to reflect your target user profile, which in your case are `vet/vet assistants`.

cjianhui commented on docs/DeveloperGuide.adoc:
```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
+* needs to search through amounts of data for information
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: all-in-one vet assistant app that can view and make appointments and search for information
```
Well done on defining a clear and specific target user profile.

cjianhui reviewed (commented):
@Aquarinte @Robert-Peng @wynonaK @chialejing 
Good job on the PR! Just some minor comments. Also, remember to update your README, User Guide and Developer Guide to reflect your product name and the respective product descriptions as well. 
Do leave this PR open as we will be tracking the milestones through this.

cjianhui reviewed (commented):
@Aquarinte @Robert-Peng @wynonaK @chialejing 
Good job on updating the About Us, README and User Guide. Do keep the PR open after reading my remarks.
Ideally, you should think through the product's end behavior and write the entire UG, for all the commands which you would want to implement. Then mark the features as future (v2.0) which you do not intend to implement in the project duration, which I see you guys have done.  Good job on that!

Finally, do ensure you update your issue tracker to reflect the user stories and tasks that you want to implement/complete during the duration of this project. You may refer to https://github.com/nus-cs2103-AY1718S2/forum/issues/28 and https://github.com/MyTAOrg/DemoGradle/issues to see how you can go about doing it. I will talk more about it during tutorial as well.

cjianhui reviewed (commented):
@Aquarinte @Robert-Peng @wynonaK @chialejing 
Good job on updating the DG to reflect your implementations as well as updating your issue tracker to reflect the different story types and tasks! It also looks like you guys are on track to complete your product! However, do remember to supplement your implementation description with diagrams like sequence or class diagrams. 
Otherwise, keep up the good work!

wynonaK committed:
Merge pull request #203 from Aquarinte/v1.5

Implement size restriction for autocomplete suggestion list and some minor fixes/improvements

Aquarinte committed:
Add 2 methods to StringUtil

Aquarinte committed:
Add descriptions for options and prefixes

Robert-Peng committed:
Update Exception

Aquarinte committed:
Fix checkstyle error

Robert-Peng committed:
Merge pull request #11 from CS2103JAN2018-F14-B2/master

Pull from team repo

Aquarinte committed:
Remove newline in StringUtil

Aquarinte committed:
Update Autocomplete.java and CommandBox.java to support descriptions for options and prefixes

Aquarinte committed:
Update and add JUnit tests for autocomplete

Robert-Peng committed:
Fix time interval to 30

Robert-Peng committed:
Set hour height even bigger

Aquarinte committed:
Help Wynona to update her collate for sampleDataUtil

wynonaK committed:
Merge pull request #197 from Robert-Peng/DisableButtons

Disable Calendar buttons & Add Exception

wynonaK committed:
Merge pull request #205 from Aquarinte/v1.5

Implement descriptions for options and prefixes in autocomplete drop down menu

Aquarinte committed:
Merge pull request #206 from Robert-Peng/IntervalBug

Interval fixed to 30

Aquarinte committed:
Merge pull request #25 from CS2103JAN2018-F14-B2/master

Pull from updated team repo (v1.5rc)

Aquarinte committed:
Remove codacy badge and glitter badge

Aquarinte committed:
Add circleci config file

Aquarinte committed:
Create project portfolio docs

Aquarinte committed:
Add new line at EOF

Robert-Peng committed:
Merge pull request #207 from Aquarinte/updateDocs

Update docs

Aquarinte committed:
Update error messages for add command

Aquarinte committed:
Modify string formatting for Person class toString() method

Aquarinte committed:
Update Nric constraints message

Aquarinte committed:
Modify string formatting for Appointment class toString() method

Aquarinte committed:
Correct error message in add command parser

Aquarinte committed:
Update and add new JUnit tests

Aquarinte committed:
Removed unused imports and add collate

Aquarinte committed:
Fix checkstyle error

chialejing committed:
Updated ParserUtil

Updated parsing for:
- Person: address
- PetPatient: species, breed, colour, blood type
- Tag

chialejing committed:
Updated constraint messages for NRIC and Tag

chialejing committed:
Fixed regression for JUnit tests

chialejing committed:
Added tests for ParserUtil

Robert-Peng committed:
minor change

Robert-Peng committed:
Update Tab pane and text colour

Robert-Peng committed:
ID

Robert-Peng committed:
Update UI color bug

Aquarinte committed:
Code improve for parserutil

Aquarinte committed:
Fix checkstyle error

Aquarinte committed:
Update and add new JUnit tests for add command

Aquarinte committed:
Fix 2 parse methods

wynonaK committed:
Merge pull request #213 from Aquarinte/v1.5

Update Add command error messages and JUnit tests

Robert-Peng committed:
changes of tab color

Aquarinte committed:
Update UG format and made some amendments

Aquarinte committed:
Merge pull request #212 from Robert-Peng/ListView

Fix List view color bug

Robert-Peng committed:
Merge pull request #214 from Aquarinte/updateDocs

Modify UG structure and made some amendments

Aquarinte committed:
Merge pull request #196 from Robert-Peng/ShowTab

Show tabs for each panelList

Aquarinte committed:
Merge pull request #26 from CS2103JAN2018-F14-B2/master

Pull from updated team repo

Robert-Peng committed:
Merge pull request #12 from CS2103JAN2018-F14-B2/master

Pull from team repo

Aquarinte committed:
Add badge for CircleCi

Aquarinte committed:
Rename labels for panels to "Contacts" and "Pet Patients"

Aquarinte committed:
Update light theme css for commandbox area

chialejing committed:
Merge branch 'master' into updateParserUtil

chialejing committed:
Update Nric.java

wynonaK committed:
Merge pull request #21 from CS2103JAN2018-F14-B2/master

Updates v1.5rc

chialejing committed:
Resolved conflicts with AddCommandSystemTest

Aquarinte committed:
Code refactoring for Autocomplete

Aquarinte committed:
Merge pull request #211 from CS2103JAN2018-F14-B2/updateParserUtil

Update ParserUtil

Aquarinte committed:
Add key to toggle autocomplete on/off

wynonaK committed:
added some minor tests.

wynonaK committed:
Added some small tests.

Aquarinte committed:
Update Ui.png, autocomplete.png and UG

Robert-Peng committed:
Update portfolio

Aquarinte committed:
Update autocomplete.png

chialejing committed:
Merge pull request #215 from Aquarinte/v1.5

Add key to toggle autocomplete on/off

chialejing committed:
Merge pull request #216 from Aquarinte/updateUI

Update UI

chialejing committed:
Added JUnit Tests for EditCommandParserTest

chialejing committed:
Removed trailing whitespaces

wynonaK committed:
refactored the find command.

wynonaK committed:
Removed unused imports.

Aquarinte committed:
Add tags for UG and updated dark theme image

Aquarinte committed:
Update DG

Aquarinte committed:
Update project portfolio

Aquarinte committed:
Fix checkstyle error

Aquarinte committed:
Fix PPP

Aquarinte committed:
Update PPP

Aquarinte committed:
Update tags

wynonaK committed:
Added tests for find command

wynonaK committed:
Fixed errors, added more tests.

chialejing committed:
Update Project Portfolio

chialejing committed:
Minor Edits to PP

chialejing committed:
Merge pull request #217 from Robert-Peng/Portfolio

Update project Portfolio page

chialejing committed:
Merge pull request #219 from wynonaK/findRefactorNewTests

Refactoring Find Command, Adding New Tests

Aquarinte committed:
Merge pull request #27 from CS2103JAN2018-F14-B2/master

Pull from updated team repo

wynonaK committed:
Merge pull request #218 from CS2103JAN2018-F14-B2/EditCommandTests

Edit Command Tests

Robert-Peng committed:
Merge pull request #13 from CS2103JAN2018-F14-B2/master

Pull from team repo

Aquarinte committed:
Update autocomplete to not show suggestions for commands that do not require prefixes and options

Aquarinte committed:
Add JUnit tests

Aquarinte committed:
Fix checkstyle error

Robert-Peng committed:
Update Portfolio details

Aquarinte committed:
Add tests to cover all relevant lines

chialejing committed:
Merge pull request #223 from Aquarinte/v1.5

Update autocomplete

chialejing committed:
Merge pull request #222 from Robert-Peng/UpdatePortfolio

Update portfolio

Aquarinte committed:
Merge branch 'master' into updateDocs

Aquarinte committed:
Merge pull request #221 from CS2103JAN2018-F14-B2/updateDocs

Update Documents

wynonaK committed:
new tests.

chialejing committed:
Merge pull request #220 from Aquarinte/updateDocs

Update docs

Robert-Peng committed:
Merge pull request #14 from CS2103JAN2018-F14-B2/master

Pull from team repo

Aquarinte committed:
Remove collate author in LightTheme css

Aquarinte committed:
Update collate files

chialejing committed:
Merge pull request #224 from Aquarinte/v1.5

Update collate files

chialejing committed:
Minor Edits

wynonaK committed:
ug changes

Aquarinte committed:
Merge pull request #225 from CS2103JAN2018-F14-B2/release1.5rc

Minor Edits

wynonaK committed:
added tags for listappt

wynonaK committed:
updated links for team

Aquarinte committed:
Merge pull request #226 from wynonaK/ugwk12

UG Updates, Project Portfolio Updates

Aquarinte committed:
Merge pull request #28 from CS2103JAN2018-F14-B2/master

Pull v1.5rc from updated team repo

Robert-Peng committed:
Change PastDateAppointment Exception into AddCommandParser.

Robert-Peng committed:
update Ui diagram, document calendarFx issue, document user warning about data integrity.

Robert-Peng committed:
Update image

chialejing committed:
Implemented Checks for Edit Command

- any appointments with concurrency, same date time or date time in the past will be checked for
- relevant error messages will be thrown out

chialejing committed:
Removed Whitespace

wynonaK committed:
added new sample data

Aquarinte committed:
Merge pull request #228 from CS2103JAN2018-F14-B2/updateEditCommand

Implemented Checks for Edit Command

Aquarinte committed:
Set light theme as default theme

wynonaK committed:
Merge pull request #227 from Robert-Peng/ExceptionUpdate

Exception update

Aquarinte committed:
Add formatting for pet patient card

wynonaK committed:
fixed indentation errors.

Robert-Peng committed:
Document calendarFX bug in UG, update Manual testing for list command

wynonaK committed:
fixed checkstyle errors

Robert-Peng committed:
commit

Aquarinte committed:
Merge pull request #229 from wynonaK/tests

Added minor tests and more Sample Data

Robert-Peng committed:
Further updates in Exception

Robert-Peng committed:
minor change

Robert-Peng committed:
Checkstyle error

Aquarinte committed:
Formatting for person card

chialejing committed:
Merge pull request #230 from Robert-Peng/ExceptionUpdate

More Exception update

Aquarinte committed:
Fix JUnit

Aquarinte committed:
Fix JUnit tests

Robert-Peng committed:
Merge pull request #231 from Aquarinte/updateUI

Update ui

Aquarinte committed:
Update messages for existing commands (clear, exit, list)

Aquarinte committed:
Update application size

Aquarinte committed:
Merge pull request #29 from CS2103JAN2018-F14-B2/master

Pull from updated team repo

Aquarinte committed:
Update messages and refactor code

Aquarinte committed:
Update messages

Aquarinte committed:
Change options to point to CliSyntax options

chialejing committed:
Merge pull request #232 from Aquarinte/v1.5

Updates for Add Command

Aquarinte committed:
Update autocomplete

Aquarinte committed:
Update gui settings

chialejing committed:
Merge pull request #233 from Aquarinte/v1.5

Update autocomplete

chialejing committed:
Merge pull request #234 from Aquarinte/updateUI

Update ui and messages for existing commands

Robert-Peng committed:
Update UserGuide about CalendarView clicking, pop out window and other "bugs".

Aquarinte committed:
Update diagrams and DG for autocomplete

Aquarinte committed:
Update DG

Aquarinte committed:
Update autocomplete DG

wynonaK committed:
Merge pull request #22 from CS2103JAN2018-F14-B2/master

Updates v1.5

wynonaK committed:
Changed as much AddressBook to Medeina, as much Owner/Person to contacts.

Aquarinte committed:
Merge pull request #236 from wynonaK/updateOwnerToContact

Update owner/person to contact, update addressbook to medeina.

wynonaK committed:
Merge pull request #23 from CS2103JAN2018-F14-B2/master

Updates

Robert-Peng committed:
Changes in UG

Robert-Peng committed:
Update Ui Class Diagram

Aquarinte committed:
Update UGDG

Aquarinte committed:
Update size for autocomplete.png

Robert-Peng committed:
Document no scroll bar in entry view.

Aquarinte committed:
Update UGDG

chialejing committed:
Updated UG

chialejing committed:
Fixed bug where history doesn't show in a section of its own

chialejing committed:
Updated Error Messages

Aquarinte committed:
Minor code update

Aquarinte committed:
Fix checkstyle

Aquarinte committed:
Merge pull request #238 from CS2103JAN2018-F14-B2/EditCommandMessage

Updated Error Messages

Robert-Peng committed:
Merge pull request #15 from CS2103JAN2018-F14-B2/master

Pull from team repo

chialejing committed:
Update Error for Edit Command

Robert-Peng committed:
Update Undo, Redo, List, Clear command and diagrams

Aquarinte committed:
Merge pull request #240 from CS2103JAN2018-F14-B2/editCmdEmergencyUpdate

Update Error for Edit Command

Robert-Peng committed:
Merge pull request #239 from Aquarinte/v1.5

Minor code update

Robert-Peng committed:
Merge pull request #237 from CS2103JAN2018-F14-B2/updateUG_v1.5_0414

Updated UG

chialejing committed:
Bug Fixed

Fixed logic error

Aquarinte committed:
Update UG CLI section and add command

Aquarinte committed:
Update UG Cli section

Robert-Peng committed:
Update diagrams

Robert-Peng committed:
Merge branch 'master' into Diagrams

wynonaK committed:
Updated the find command error.

Updated DG for ListAppointment and Delete.

Updated Glossary of terms.

Updated Major and Minor enhancements for myself.

Aquarinte committed:
Update UG

Aquarinte committed:
Merge pull request #235 from Robert-Peng/UpdateDGPort

Update DG UG Portfolio

Robert-Peng committed:
Minor update

Robert-Peng committed:
Merge branch 'Diagrams' of https://github.com/Robert-Peng/main into Diagrams

Aquarinte committed:
Fix checkstyle, update introduction, update autocomplete to show max size

Aquarinte committed:
Upload colours.png for introduction

Robert-Peng committed:
Remove black line at the end of SS

Aquarinte committed:
Update UG command syntax image and some phrasing

Aquarinte committed:
Merge branch 'master' into updateDocs

Aquarinte committed:
Update image size for colours.png in UG

Aquarinte committed:
Resize colours.png to 300 in UG

Aquarinte committed:
Bold a word

Aquarinte committed:
Update example for add

Robert-Peng committed:
pet patient

Aquarinte committed:
Merge pull request #241 from CS2103JAN2018-F14-B2/editCmdEmergencyUpdate

Bug Fixed

Aquarinte committed:
Made changes to DG based on comment

Aquarinte committed:
Merge pull request #242 from Robert-Peng/Diagrams

Update UG DG

Robert-Peng committed:
Update image sizes

Robert-Peng committed:
Merge pull request #244 from Aquarinte/updateDocs

Update docs

wynonaK committed:
Updates to changes requested.

Robert-Peng committed:
Minor update

wynonaK committed:
Merge branch 'master' into dgUpdatesListApptDelete

Robert-Peng committed:
Update image sizes

Robert-Peng committed:
Change size

Robert-Peng committed:
Change size

chialejing committed:
Merge pull request #243 from wynonaK/dgUpdatesListApptDelete

Developers Guide Updates

wynonaK committed:
Merge pull request #24 from CS2103JAN2018-F14-B2/master

Updates

chialejing committed:
Merge pull request #245 from Robert-Peng/Diagrams

Update image sizes

Robert-Peng committed:
Merge pull request #16 from CS2103JAN2018-F14-B2/master

Pull from team repo

Aquarinte committed:
Merge pull request #30 from CS2103JAN2018-F14-B2/master

Pull from updated team repo

Robert-Peng committed:
Update Portfolio

chialejing committed:
Updated DG and Diagrams

Aquarinte committed:
Updated project portfolio page

chialejing committed:
Added spacing after figure

Robert-Peng committed:
Update pic name

Robert-Peng committed:
Image size

Robert-Peng committed:
Ui pic size

chialejing committed:
Update UG Examples for Edit

chialejing committed:
Minor Edit to UG

chialejing committed:
Trying to shorten PPP

chialejing committed:
Still trying to shorten PPP...

wynonaK committed:
Update DG for find command.

wynonaK committed:
Update PPP

Robert-Peng committed:
Update Portfolio

chialejing committed:
Fix for wrong message shown

Robert-Peng committed:
Collate

wynonaK committed:
Updated user guide

wynonaK committed:
Fixed bug in parser util.

wynonaK committed:
updated positions for PPP.

Aquarinte committed:
Update UG DG and PPP

wynonaK committed:
Fixed failing test cases.

Aquarinte committed:
Fix trailing whitespace

chialejing committed:
Updated EditCommandSystemTest

Aquarinte committed:
Move the undo/redo section down. Update DG

Aquarinte committed:
Merge pull request #249 from CS2103JAN2018-F14-B2/fixForEditCommand

Fix + Added JUnit Test for EditCommand

Aquarinte committed:
Merge pull request #247 from CS2103JAN2018-F14-B2/updateDG_20180415_1938

Updated DG and Diagrams

Robert-Peng committed:
collate code

Aquarinte committed:
Update UG PPP

Robert-Peng committed:
EOF

Aquarinte committed:
Remove -fa option

Aquarinte committed:
Remove option -fa

chialejing committed:
Merge pull request #246 from Robert-Peng/UpdateDG

Minor updates on docs

Aquarinte committed:
Update autocomplete png

wynonaK committed:
Updates

Aquarinte committed:
Merge branch 'master' into v1.5

wynonaK committed:
no message

wynonaK committed:
no message

Robert-Peng committed:
Merge pull request #248 from Aquarinte/v1.5

Update Docs

chialejing committed:
Collate v1.5

wynonaK committed:
Collate

wynonaK committed:
Merge branch 'master' into updateDGFind

Robert-Peng committed:
Merge pull request #251 from CS2103JAN2018-F14-B2/collate1.5

Collate v1.5

wynonaK committed:
Update DeveloperGuide.adoc

wynonaK committed:
Update DeveloperGuide.adoc

chialejing committed:
Merge pull request #250 from wynonaK/updateDGFind

Update DG and UG, as well as fixed error for date parser.

Aquarinte committed:
Update UserGuide.adoc
